PKGBUILD

```diff
@@ -11,7 +11,7 @@ license=(MIT custom)
 depends=(obs-studio opencv)
 makedepends=(cmake)
 source=("${_bgdir}.tar.gz::https://github.com/royshil/$pkgname/archive/refs/tags/v${pkgver//_/-}.tar.gz"
-        "https://github.com/microsoft/onnxruntime/releases/download/v1.7.0/$_ONNXRUNTIME_FILE")
+        "https://github.com/microsoft/onnxruntime/releases/download/v${_onnxversion}/onnxruntime-linux-x64-${_onnxversion}.tgz")
```

Comment:
why not reusing `$_onnxdir` instead of `onnxruntime-linux-x64-${_onnxversion}`?

Reply:
Because I perceive a difference between a file name and a directory name even if the file name happens to be {directory name}.{extension} – but it’s up to you :-)

Reply:
Let’s continue this in #2 – sorry about the momentary confusion on the spelling of ONNX.